citestwheel.Dockerfile

```
@@ -4,7 +4,7 @@ ARG LINUX_VER=notset
 ARG BASE_IMAGE=nvcr.io/nvidia/cuda:${CUDA_VER}-devel-${LINUX_VER}
 ARG AWS_CLI_VER
```

Same suggestion as above

```suggestion
ARG AWS_CLI_VER=notset
```